frontend/common/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { DetailedHTMLProps } from "react";
+
+const DismissBtn: React.FC<
```

Comment:
Should the default DismissBtn have some additional styling? Or use our Button component?

Reply:
Ah well, I tried to reuse an existing component from the commit history but I'm not sure I want to spend time making it work for this situation.  This message doesn't even have a dismiss button in the design.  I've removed the Alert component and just used a regular div instead.